services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/CongestionTest.java

```diff
@@ -0,0 +1,49 @@
+package com.mapbox.api.directions.v5.models;
+
+import com.mapbox.core.TestUtils;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class CongestionTest extends TestUtils {
+
+  @Test
+  public void sanity() {
+    assertNotNull(getDefault());
+  }
+
+  @Test
+  public void testSerializable() throws Exception {
```

Comment:
Minor thing: We should remove `test` as it's implicit from `@Test`. Could we add a better naming reflecting what's being tested here?

Reply:
it's [common naming](https://github.com/mapbox/mapbox-java/search?q=%22public+void+testSerializable%28%29+throws+Exception%22) for our tests